torch/ao/quantization/fx/prepare.py

```diff
@@ -129,6 +130,31 @@
     "output_act_obs_or_fq_ctr": torch.ao.quantization.qconfig._default_quint8_placeholder_qconfig.activation
 }
 
+def _needs_obs_or_fq(prev_output_dtype, prev_output_is_dynamic, cur_target_dtype, cur_target_is_dynamic, reuse_input_obs_or_fq):
```

Comment:
Can you just pass in the nodes?

Reply:
probably can't do this for output I think, since we didn't get `prev_output_dtype`, `prev_output_is_dynamic` from node.meta, it's hardcoded instead

Reply:
I see